assets/css/styles.css

```
@@ -343,3 +343,6 @@
     top: 0px;
   }
 }
+.formContainer {
+  background-color: blue;
+}
```

Add a new line after this @AlizayAyesha if not the last line '}' with this will be removed after merging
```suggestion
}

```